src/main/scala/org/allenai/scholar/StringUtils.scala

```diff
@@ -21,6 +21,8 @@ object StringUtils {
 
     def removePunctuation(): String = punctuationRegex.replaceAllIn(str, " ")
 
+    def collapseWhitespace(): String = whiteSpaceRegex.replaceAllIn(str, " ")
```

Comment:
We actually have equivalent logic in the `org.allenai.scholar` package object's `normalizeText` but I thought this was a better home.